pages/api/trips/index.js

```diff
@@ -0,0 +1,23 @@
+import dbConnect from "@/db/connect";
+import Trip from "@/db/models/Trips";
+
+export default async function handler(request, response) {
+  await dbConnect();
+
+  try {
+    if (request.method === "GET") {
+      const trips = await Trip.find();
+      return response.status(200).json(trips);
```

Comment:
you can write the response sending in the same line as the return statement. But this can be misleading. Some might think that the thing that the handler function returns is sent back to the client. But this is not the case! For clarity you should separate the "early return" and the response visually:
```suggestion 
      response.status(200).json(trips);
      return
```